clear/bgp_frr_v4.py

```diff
@@ -0,0 +1,200 @@
+import click
+from clear.main import *
+
+
+###############################################################################
+#
+# 'clear bgp' cli stanza
+#
+###############################################################################
+
+
+@cli.group(cls=AliasedGroup, default_if_no_args=True)
+def bgp():
+    """Clear BGP peers / state"""
+    pass
+
+
+# Default 'bgp' command (called if no subcommands or their aliases were passed)
+@bgp.command(default=True)
+def default():
+    """Clear all BGP peers"""
+    command = 'sudo vtysh -c "clear bgp *"'
+    run_command(command)
+
+
+@bgp.group(cls=AliasedGroup, default_if_no_args=True, context_settings=CONTEXT_SETTINGS)
+def neighbor():
+    pass
+
+
+@neighbor.command(default=True)
+@click.argument('ipaddress', required=False)
+def default(ipaddress):
+    """Clear BGP neighbors"""
+    if ipaddress is not None:
+        command = 'sudo vtysh -c "clear bgp {} "'.format(ipaddress)
+    else:
+        command = 'sudo vtysh -c "clear bgp *"'
+    run_command(command)
+
+
+# 'in' subcommand
+@neighbor.command('in')
+@click.argument('ipaddress', required=False)
+def neigh_in(ipaddress):
+    if ipaddress is not None:
```

Comment:
Many of these commands and groups are missing docstrings. The Click library uses the docstrings to present usage information to the user as part of the 'help' text.

Could you please go through all files and add all missing docstrings; just a brief description of each group/command?

Test each group/command by using the `--help` flag.